modules/nodes-scheduler-taints-tolerations-about.adoc

```diff
@@ -33,7 +33,7 @@ spec:
 ....
   template:
 ....
-    spec
+    spec:
```

Comment:
@mburke5678, do you have anyone in mind from QE that can confirm this YAML update? Thanks!